models/issue.go

```diff
@@ -504,57 +504,58 @@ func Issues(opts *IssuesOptions) ([]*Issue, error) {
 	}
 
 	sess := x.Limit(setting.IssuePagingNum, (opts.Page-1)*setting.IssuePagingNum)
+	sess.Alias("issue")
```

Comment:
This explicitly sets the table alias rather than letting xorm decide whether it's necessary (see my previous comment on why I think it's good to do this).